### PR TITLE
productDetail 

### PR DESCRIPTION
**Pull request by guilhermemamprin**
Implements #20

### CODE REVIEW

File: src/app/controllers/products/productDetail.controller.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+module webShoes {
+  'use strict';
+  class Product {
+    id         : number;
+    img1       : string;
+    img2       : string;
+    img3       : string;
+    img4       : string;
+    manufactor : string;
```
Comment (lucasBgabriel) [inline src/app/controllers/products/productDetail.controller.ts:9]: I guess this variable should be manufacturer

**Comment by lucasBgabriel** [comment]
👍

**Comment by lucasBgabriel** [comment]
Don't forget to rebase before merge

File: .idea/libraries/webshoes_frontend_node_modules.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<component name="libraryTable">
+  <library name="webshoes-frontend node_modules" type="javaScript">
+    <properties>
+      <option name="frameworkName" value="node_modules" />
```
Comment (guilhermemamprin) [inline .idea/libraries/webshoes_frontend_node_modules.xml:4]: @adrianovalente wtf is this? hahah

**Pull request merged**